auto_round/mllm/eval.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import tabulate
+from functools import partial
+
+import pandas as pd
+from vlmeval.vlm import *
+from vlmeval.api import *
+from vlmeval.config import supported_VLM
+from vlmeval.dataset import build_dataset
+from vlmeval.inference import infer_data_job
```

Comment:
I think you should use lazy import or some other ways. Otherwise exception will raise if vlmeal has not been installed.